test-jest/shape/Trapezoid.spec.tsx

```diff
@@ -1,38 +1,36 @@
 import React from 'react';
-import { expect } from 'chai';
-// eslint-disable-next-line import/no-extraneous-dependencies
-import { Surface, Trapezoid } from 'recharts';
-import { render } from 'enzyme';
+import { render } from '@testing-library/react';
+import { Surface, Trapezoid } from '../../src';
 
 describe('<Trapezoid />', () => {
-  it('Render 1 Trapezoid in simple Trapezoid', () => {
-    const wrapper = render(
+  test('Render 1 Trapezoid in simple Trapezoid', () => {
+    const { container } = render(
       <Surface width={500} height={500}>
         <Trapezoid fill="#f00" x={300} y={100} upperWidth={100} lowerWidth={0} height={50} />
       </Surface>,
     );
 
-    expect(wrapper.find('.recharts-trapezoid').length).to.equal(1);
+    expect(container.querySelectorAll('.recharts-trapezoid')).toHaveLength(1);
   });
 
-  it('Render 1 Trapezoid with animation in simple Trapezoid', () => {
-    const wrapper = render(
+  test('Render 1 Trapezoid with animation in simple Trapezoid', () => {
+    const { container } = render(
       <Surface width={500} height={500}>
         <Trapezoid fill="#f00" x={300} y={100} upperWidth={100} lowerWidth={0} height={50} isUpdateAnimationActive />
```

Comment:
More curious than anything - is there anything we can do to test `isUpdateAnimationActive`? Just a thought, non-blocking for the migration